websites/A/ANIMETV7/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "1105891230191206432",
+		"name": "ep_1073"
+	},
+	"service": "ANIMETV7",
+	"description": {
+		"en": "ANIMETV7 is a free Anime and Manga streaming website with AniList integration."
+	},
+	"url": "animetv7.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/CtyAaoq.jpg",
```

Comment:
Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).